apps/meteor/app/assets/server/assets.ts

```diff
@@ -333,37 +355,37 @@ function addAssetToSetting(asset, value) {
 			asset,
 			public: true,
 			wizard: value.wizard,
-		},
+		} as Record<string, any>,
```

Comment:
any special reason that you had to add this? is complaining some options are not in `ISettingAddOptions`? maybe the best should be adding them there.